src/components/button/_macro.njk

```diff
@@ -64,7 +64,7 @@
         {% endif %}
     {% endif %}
 
-    {% set tag = "a" if params.url or params.dsExample else "button" %}
```

Comment:
Do we even need this if it wasn't being used in any examples? It means now that all the button examples are producing invalid HTML by having attributes like `type` on an hyperlinks e.g.
```
<a type="submit" class="ons-btn">
        <span class="ons-btn__inner"><span class="ons-btn__text">Save and continue</span>
            </span>
</a>
```

Reply:
I think we just add `onsubmit="return false;"` or `action="#0"` on all the forms in examples instead